sdk/client/metric_reporter.go

```diff
@@ -136,6 +137,27 @@ func (r *metricReporter) WithDialOptions(options ...grpc.DialOption) Client {
 	return r
 }
 
+func (r *metricReporter) WithProtoBackoffSettings(backoffSettings *proto.Backoff) Client {
+
```

Comment:
unnecessary line again

Reply:
since this code is the same, is it possible to move them to the one place? https://github.com/nginx/agent/pull/361/files#diff-6dfc26afa255de7757f305a8a305956ab624401f6a0e19597faf9d3dd4afab80R124

Reply:
Maybe instead of adding the new function `WithProtoBackoffSettings` you could add a helper function in here https://github.com/nginx/agent/blob/main/sdk/agent/config/config_helpers.go to convert the `*proto.Backoff` to the `backoff.BackoffSettings` which you can call before you call the function `WithBackoffSettings` .